components/language-switcher/language-switcher.html

```diff
@@ -0,0 +1,33 @@
+<form class="mzp-c-language-switcher " method="get"
+ action="#">
+  <a class="mzp-c-language-switcher-link" href="https://www.mozilla.org/locales/">Language</a>
+  <label
+   for="mzp-c-language-switcher-select">Language</label>
+  <select id="mzp-c-language-switcher-select"
+   class="mzp-js-language-switcher-select" name="lang">
+    <option value="en-US">English (US)</option>
+    <option value="de">Deutsch</option>
+    <option value="fr">Français</option>
+    <option value="es-ES">Español</option>
+    <option value="ja">日本語</option>
+  </select>
+  <!-- form submit button is shown when JavaScript is not enabled -->
+  <button type="submit">Go</button>
+</form>
+
+<script
+ src="../../../assets/protocol/protocol/js/protocol-lang-switcher.js">
```

Comment:
This path should be:
```suggestion
 src="{{ '/protocol/js/protocol-lang-switcher.js' | path }}">
```